main/tray.ts

```diff
@@ -39,17 +50,42 @@ export const resetTray = () => {
   }
 
   tray.removeAllListeners('click');
+  tray.removeAllListeners('right-click');
 
   tray.setImage(path.join(__dirname, '..', 'static', 'menubarDefaultTemplate.png'));
   tray.on('click', openCropperWindow);
   tray.on('right-click', openContextMenu);
 };
 
-export const setRecordingTray = (stopRecording: () => void) => {
+export const setRecordingTray = () => {
   animateIcon();
 
+  tray.removeAllListeners('right-click');
+
   // TODO: figure out why this is marked as missing. It's defined properly in the electron.d.ts file
-  tray.once('click', stopRecording);
+  tray.once('click', onRecordingTrayClick);
+  tray.on('right-click', openRecordingContextMenu);
+};
+
+export const setPausedTray = () => {
+  if (trayAnimation) {
+    clearTimeout(trayAnimation);
+  }
+
+  tray.removeAllListeners('right-click');
+
+  tray.setImage(path.join(__dirname, '..', 'static', 'pauseTemplate.png'));
```

Comment:
i created a new pause icon, hopefully it looks good enough? 😅 lemme know if I should change it or whatever